docs/faq/migration/token-migration.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: 'Token Migration'
+sidebar_position: 1
+---
+
+# Token Migration
+
+### How can I migrate LYXe to LYX?
+
+You can migrate LYXe to LYX using the official [Migration Bridge](https://migrate.lukso.network/). Please check the `lukso.network` domain in order to guarentee that you are on the official app, before connecting your wallet.
+
+### How long will the LYXe migration be open?
+
+The migration process started on the 4th of July, 2023, and is currently planned to stay open until July 2027 (4 years), allowing anyone to migrate. The Foundation for the New Creative Economies can pause the migration bridge. Please refer to the official announcements from the [LUKSO Twitter Account](https://twitter.com/lukso_io).
+
+### How long does it take to migrate my LYXe?
+
+The token migration usually takes about 15min, as it is built for security. Sometimes, it could take multiple hours if the bridging services are restarted or redeployed after maintenance.
+
+### Will LYXe be migrated automatically?
+
+LYXe will not be migrated automatically, you have to instanciate it manually. There is likely an amount of LYXe that will never migrate due to lost wallets. Some people might even keep some LYXe for sentimental reasons.
+
+### What happens with LYXe being sent back to the LYXe Contract?
+
+Any LYXe accidentally sent to the [LYXe Token address](https://etherscan.io/token/0xA8b919680258d369114910511cc87595aec0be6D) on Ethereum will be automatically sent as LYX to the sending address on the LUKSO Blockchain.
```

Comment:
wait what? Not true. please remove

Reply:
wait what? True. please read https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5

Reply:
added the link, so everyone can verify

Reply:
Ok my bad I didn't know it was public yet ;)